File: services/export_service.py
"""Serviço de exportação — gera o CSV com as 12 parcelas anuais (RF12)."""

from models.orcamento import Orcamento


class ExportService:
    @staticmethod
    def gerar_csv(orcamento: Orcamento, arquivo: str) -> None:
        raise NotImplementedError

File: src/services/ExportService.ts
/** Serviço de exportação — gera o CSV com as 12 parcelas anuais (RF12). */
import type { Orcamento } from "../models/Orcamento.js";

export class ExportService {
  static gerarCsv(orcamento: Orcamento, arquivo: string): void {
    orcamento.exportarCsv(arquivo);
  }
}
